src/msm.rs

```diff
@@ -529,28 +630,33 @@ mod test {
         }
     }
 
+    #[cfg(test)]
```

Comment:
Hmm, I think this is unnecessary. 
This `rum_msm_cross` is already inside the `test`module.